umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/echo10.clj

```diff
@@ -43,3 +47,66 @@
+                                                                                :Format "XLS, PDF, PNG"}
+                                                                               {:FormatType "Native",
+                                                                                :Format "HTML"}]}
+        actual-multi-dataformat-without-price (-> base-record
+                                                  (string/replace #"<Price>0</Price>" ""))
+        expected-single-dataformat-with-price {:FileDistributionInformation [{:FormatType "Native",
+                                                                              :Fees "0",
+                                                                              :Format "XLS, PDF, PNG"}]}
+        actual-single-dataformat-with-price (-> base-record
+                                                (string/replace #"<DataFormat>HTML</DataFormat>" ""))
+        expected-multi-dataformat-with-price {:FileDistributionInformation [{:FormatType "Native",
+                                                                             :Fees "0",
+                                                                             :Format "XLS, PDF, PNG"}
+                                                                            {:FormatType "Native",
+                                                                             :Fees "0",
+                                                                             :Format "HTML"}]}
+        expected-price-without-dataformat {:FileDistributionInformation [{:FormatType "Native",
+                                                                          :Fees "0",
+                                                                          :Format "Not provided"}]}
+        actual-price-without-dataformat (-> base-record
+                                            (string/replace #"<DataFormat>.*</DataFormat>" ""))
+        actual-no-price-no-dataformat (-> base-record
+                                          (string/replace #"<Price>.*</Price>" "")
+                                          (string/replace #"<DataFormat>.*</DataFormat>" ""))]
+
+    (are3 [expected-result test-string]
+      (is (= expected-result
+             (echo10/parse-archive-dist-info test-string)))
+
+      "Single DataFormat test without Price"
+      expected-single-dataformat-without-price
+      actual-single-dataformat-without-price
+
+      "Multi DataFormat test without Price"
+      expected-multi-dataformat-without-price
+      actual-multi-dataformat-without-price
+
+      "Single DataFormat test with Price"
+      expected-single-dataformat-with-price
+      actual-single-dataformat-with-price
+
+      "Multi DataFormat test with Price"
+      expected-multi-dataformat-with-price
+      base-record
+
+      "Price without DataFormat"
+      expected-price-without-dataformat
+      actual-price-without-dataformat
+
+      "Price without DataFormat"
```

Comment:
Change this doc string.

Reply:
done.  This test is for when there are no Price or DataFormat elements.

Reply:
done.  This test is for when there are no Price or DataFormat elements.